### ISSUE TITLE
Volume controls not working after 0.92

### ISSUE THREAD

**Issue by caso1982** [custom-components/media_player.braviatv_psk#10]
Was working faultlessly before updating to HA 0.92. 
Initially needed to add the `_init_.py` file to `config/custom_components/braviatv_psk` to get the component working again. 
All other controls are working except for volume up and volume down controls. 

![Screen Shot 2019-04-29 at 3 43 05 pm](https://user-images.githubusercontent.com/46815185/56882204-86750c80-6a95-11e9-8c8a-540198b6a666.png)

### Config

```
media_player:
  - platform: braviatv_psk
    name: Theatre TV Sony
    host: !secret sony_psk_ip
    psk: !secret sony_psk
    mac: !secret sony_mac
    amp: true
```

**Comment by gerard33** [comment]
I can confirm this. Will have a look at it.

**Comment by gerard33** [comment]
Fixed in 0.92.1 -> https://github.com/home-assistant/home-assistant/pull/23419
